### ISSUE TITLE
Pool Visibility Feature

### ISSUE THREAD

**Issue by jadasi** [OverstepCo/fun-sports-pools-admin#5]
Pools need to have different states of visibility:
- unpublished (saved but not visible to public)
- published (saved and visible)
- live (published and ongoing)
- finished (published and 24 hours after going live)

 _note: states 'live' and 'finished' are only on **published** pools._

If a pool is published, admins should be able to unpublish it at **any** time.

**Comment by AreYouAHacker** [comment]
@jadasi should the live state be set automatically when the start date is reached? Or should the admin set it? Same thing for the finished state?

**Comment by jadasi** [comment]
Yes. It will automatically go live once the start date is reached. It will only go live though if it is published. Same thing for the finished date.
